sdk/ai/ai-inference-rest/src/models.ts

```diff
@@ -191,17 +202,17 @@ export type ChatCompletionsToolDefinition =
 export type ChatCompletionsNamedToolSelection =
   | ChatCompletionsNamedToolSelectionParent
   | ChatCompletionsNamedFunctionToolSelection;
-/** Alias for UnknownParams */
-export type UnknownParams = string | "error" | "drop" | "pass_through";
+/** Alias for ExtraParameters */
+export type ExtraParameters = string;
 /** Alias for ChatRole */
-export type ChatRole = string | "system" | "user" | "assistant" | "tool";
+export type ChatRole = string;
 /** Alias for ChatMessageImageDetailLevel */
-export type ChatMessageImageDetailLevel = string | "auto" | "low" | "high";
+export type ChatMessageImageDetailLevel = string;
 /** Alias for ChatCompletionsResponseFormat */
-export type ChatCompletionsResponseFormat = string | "text" | "json_object";
+export type ChatCompletionsResponseFormat = string;
 /** Alias for ChatCompletionsToolSelectionPreset */
-export type ChatCompletionsToolSelectionPreset =
-  | string
-  | "auto"
-  | "none"
-  | "required";
+export type ChatCompletionsToolSelectionPreset = string;
+/** Alias for EmbeddingEncodingFormat */
```

Comment:
Why were the "auto", "none" and "required" options dropped here?

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-js/pull/30379/commits/74fe10df32a86bd41c499f0e84ba2363df5ee59b)